_posts/2024-04-24-highlights-otp-27.md

```diff
@@ -842,6 +843,43 @@ true
 [{a,true}]
 ```
 
+# New SSL client-side stapling support
+
+A new feature in the SSL client in Erlang/OTP 27 is support for [OCSP
+stapling](https://en.wikipedia.org/wiki/OCSP_stapling) for easier and
+faster verification of the revocation status of server
+certificates.
+
+With OCSP stapling, the SSL client can streamline the validation of
+revocation status. Normally the client would have to query the
+[CA (Certificate
+Authority)](https://en.wikipedia.org/wiki/Certificate_authority) using
+[OCSP (Online Certificate Status
+Protocol)](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol)
+to ensure that the server's certificate has not been
+[revoked](https://en.wikipedia.org/wiki/Certificate_revocation).
+
+The basic idea behind OCSP stapling is that the server itself will
+proactively query the CA regarding the revocation status for its own
+certificate and "staple" the time-stamped OCSP response from the CA to
+the certificate. When a client connects, the server passes along
+its OCSP-stapled certificate to the client. To verify the revocation
+status, the client only needs to check that the OCSP response was
+signed by the CA.
+
+Here follows an example showing how OCSP stapling can be enabled in the
+SSL client:
+
+```erlang
+1> ssl:start().
+ok
+2> {ok, Socket} = ssl:connect("duckduckgo.com", 443,
+                              [{cacerts, public_key:cacerts_get()},
+                               {stapling, #{ocsp_nonce => false}}]).
```

Comment:
`{stapling, #{ocsp_nonce => false}}` can be replaced with simpler
`{stapling, staple}`.

It is expected to work on master when this PR https://github.com/erlang/otp/pull/8425 is merged (accepted by @IngelaAndin already)

Reply:
Thanks, I have updated the example.